homeassistant/components/surepetcare/__init__.py

```diff
@@ -62,6 +66,13 @@
     extra=vol.ALLOW_EXTRA,
 )
 
+LOCK_STATE_SERVICE_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_FLAP_ID): cv.positive_int,
```

Comment:
We only use `CONF_*` constants for configuration items. For service items we use `ATTR_*`.

Reply:
If we move the service schema definition inside `async_setup` would it be possible to validate the flap id among existing flap ids?

Reply:
Renamed `CONF_FLAP_ID` and `CONF_LOCK_STATE` to `ATTR_`, moved schema definition inside `async_setup` and validated that `flap_id` is one of the ids provided in the configuration